lib/express-handlebars.js

```diff
@@ -190,11 +188,19 @@ ExpressHandlebars.prototype.render = function (filePath, context, options) {
 };
 
 ExpressHandlebars.prototype.renderView = function (viewPath, options, callback) {
-    var context = options,
-        data    = options.data;
+    options || (options = {});
+
+    var context = options;
+    var data    = options.data;
 
-    var helpers = utils.extend({},
-            this.handlebars.helpers, this.helpers, options.helpers);
```

Comment:
I removed the merge of `this.handlebars.helpers` since Handlebars will do this merge automatically at render time.